e2e/pod.go

```diff
@@ -298,7 +298,7 @@ func deletePod(name, ns string, c kubernetes.Interface, t int) error {
 		}
 		e2elog.Logf("%s app  to be deleted (%d seconds elapsed)", name, int(time.Since(start).Seconds()))
 		if err != nil {
-			return false, err
+			return false, fmt.Errorf("failed to get pod: %w", err)
```

Comment:
@yati1998 we are interchangeably using `app` and `pod`, I would suggest to keep it in a consistent way like `app`